app/src/androidTest/java/com/hedvig/app/feature/marketpicker/MarketPickerActivityTest.kt

```diff
@@ -97,15 +97,111 @@ class MarketPickerActivityTest {
         val market = pref.getString(Market.MARKET_SHARED_PREF, null)
         val language = pref.getString(SettingsActivity.SETTING_LANGUAGE, null)
 
-        Assert.assertThat(market, CoreMatchers.containsString(MARKET))
-        Assert.assertThat(language, CoreMatchers.containsString(Language.SETTING_SV_SE))
+        assertThat(market).isEqualTo("SE")
+        assertThat(language).isEqualTo(Language.SETTING_SV_SE)
+    }
+
+    @Test
+    fun noPreselectedMarket() {
```

Comment:
Can we make these test names slightly more descriptive?

Perhaps something like `shouldNotPreselectMarketWhenGeoOutsideAvailableMarkets`